### DISCUSSION TITLE
Modeling Motion of Anchored Buoy?

### DISCUSSION THREAD

**Discussion by LaunaG** [Parcels-code/Parcels#1265] [Q&A]
Hello!  I would like to create a Kernel that simulates the motion of a buoy that is anchored to the bottom of the ocean with a rope.  Does anyone know of past attempts made to accomplish this, or have suggestions for simple approaches or physics equations to consider?

**Comment by erikvansebille** [comment]
Oh wow, that is an ambitious project. I do not know of anyone who has implemented this in Parcels; but if you have an equation then it shouldn't be that difficult? Let me know if I can help
